netflix-assignment/src/app/(route)/home/_components/Preview.tsx

```diff
@@ -0,0 +1,50 @@
+"use client"
+import { useEffect } from 'react';
+import { useContentStore } from '../../../../store/useContentStore';
+import { Movie } from '../../../../types/movie';
+import { TvShow } from '@/types/tvshows';
```

Comment:
types.ts 안에 Movie와 TvShow를 모두 정의해서 각각 export 하고
```suggestion
import { useContentStore } from '@/store/useContentStore';
import { Movie, TvShow } from '@/types/types';
```
와 같이 써도 깔끔했겠다는 생각이 드네욥 😉